rust/kcl-lib/src/execution/exec_ast.rs

```diff
@@ -2239,7 +2240,8 @@ impl FunctionSource {
                             arg.value = arg
                                 .value
                                 .coerce(
-                                    &RuntimeType::from_parsed(ty.inner.clone(), exec_state, arg.source_range).unwrap(),
+                                    &RuntimeType::from_parsed(ty.inner.clone(), exec_state, arg.source_range)
+                                        .map_err(|e| KclError::Semantic(e.into()))?,
```

Comment:
Did you hit these unwraps?

Reply:
Yes, one of them was hit. That's why I fixed them to not unwrap.

Reply:
Hmm, that might be a bug somewhere else. or it might just be bad handling of `any`

Reply:
Looks like missing handling of `any`. `PrimitiveType::Any` was never implemented. I'm on it.